Sources/ArcGISToolkit/Components/JobManager/JobManager.swift

```diff
@@ -0,0 +1,305 @@
+/// is backgrounded so that jobs that are not yet started on the server, can have some time to
+/// allow them to start. This means if you kick off a job and it hasn't actually started on the server
+/// when the app is backgrounded, the job should have enough time to start on the server which
+/// will cause it to enter into a polling state. When the job reaches the polling state the
+/// status of the work on the server can be checked intermittently.
+///
+/// To enable polling while an app is backgrounded, the job manager will request from the system a
+/// background refresh task (if enabled via the ``JobManager/preferredBackgroundStatusCheckSchedule``
+/// property). If the system later executes the background refresh task then the
+/// job manager will check the status of any running jobs. At that point the jobs may start
+/// downloading their result. Note, this does not work on the simulator, this behavior can only
+/// be tested on an actual device.
+///
+/// Now that the job can check status in the background, it can start downloading in the background.
+/// By default, jobs will download their results with background URL session. This means that the
+/// download can execute out of process, even if the app is terminated. If the app is terminated and
+/// then later relaunched by the system because a background downloaded completed, then you may
+/// call the ``JobManager/resumeAllPausedJobs()`` method from the application relaunch point,
+/// which will correlate the jobs to their respective downloads that completed and the jobs will
+/// then finish. The app relaunch point can happen via the SwiftUI modifier `.backgroundTask(.urlSession(...))`.
+/// In UIKit it would be the `UIApplicationDelegate` method `func application(UIApplication, handleEventsForBackgroundURLSession: String, completionHandler: () -> Void)`
+/// - Since: 200.3
+@MainActor
+public class JobManager: ObservableObject {
+    /// The shared job manager.
+    public static let `shared` = JobManager()
+    
+    /// The jobs being managed by the job manager.
+    @Published
+    public var jobs: [any JobProtocol] = []
+    
+    /// The key for which state will be serialized under the user defaults.
+    private var defaultsKey: String {
+        return "com.esri.ArcGISToolkit.jobManager.jobs"
+    }
+    
+    /// The preferred schedule for performing status checks while the application is in the
+    /// background. This allows an application to check to see if jobs have completed and optionally
+    /// post a local notification to update the user. The default value is `disabled`.
+    /// When the value of this property is not `disabled`, this setting is just a preference.
+    /// The operating system ultimately decides when to allow a background task to run.
+    /// If you enable background status checks then you must also make sure to have enabled
+    /// the "Background Fetch" background mode in your application settings.
```

Comment:
Not a big deal but it seems like "Fetch" is normally lowercased.

<img width="616" alt="Screenshot 2023-09-06 at 10 17 38" src="https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/16397058/0e6411c3-7de4-463c-965c-a7293b4eb1bb">

```suggestion
    /// the "Background fetch" background mode in your application settings.
```